sklearn/utils/estimator_checks.py

```diff
@@ -70,6 +70,13 @@
                 'OrthogonalMatchingPursuit', 'PLSCanonical', 'PLSRegression',
                 'RANSACRegressor', 'RadiusNeighborsRegressor',
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
+# Outlier detection estimators
+OUTLIER_DETECTION = ['EllipticEnvelope', 'OneClassSVM',
+                     'LocalOutlierFactor', 'IsolationForest']
```

Comment:
you did not add OutlierMixin to OneClassSVM why?

Reply:
Why do we need this list beyond _estimator_type?